/jds/bin/diff -uprN slocate-3.1.old/doc/slocate.1 slocate-3.1/doc/slocate.1
--- slocate-3.1.old/doc/slocate.1	2009-02-19 18:56:27.152048000 +0000
+++ slocate-3.1/doc/slocate.1	2009-02-19 18:59:02.676293000 +0000
@@ -1,3 +1,4 @@
+'\" te
 .TH SLOCATE 1 "January 6, 2006" 
 .SH NAME
 slocate \- Security Enhanced version of the GNU Locate
@@ -109,5 +110,21 @@ Kevin Lindsay - Copyright (c) 1999, 2000
 Report bugs to slocate@trakker.ca
 .SH URL
 http://slocate.trakker.ca/
+
+.SH "ATTRIBUTES"
+.PP
+See \fBattributes\fR(5)
+for descriptions of the following attributes:
+.sp
+.TS
+tab(^G) allbox;
+cw(2.750000i)| cw(2.750000i)
+lw(2.750000i)| lw(2.750000i).
+ATTRIBUTE TYPE^ATTRIBUTE VALUE
+Availability^SUNWslocate
+Interface stability^Volatile
+.TE
+.sp
 .SH "SEE ALSO"
 \fBupdatedb\fP(1)
+...\" LSARC 2008/447 slocate for OPenSolaris
/jds/bin/diff -uprN slocate-3.1.old/doc/updatedb.1 slocate-3.1/doc/updatedb.1
--- slocate-3.1.old/doc/updatedb.1	2009-02-19 18:56:27.151886000 +0000
+++ slocate-3.1/doc/updatedb.1	2009-02-19 18:59:10.009775000 +0000
@@ -1,3 +1,4 @@
+'\" te
 .TH UPDATEDB 1 \" -*- nroff -*-
 .SH NAME
 updatedb \- update the slocate database
@@ -64,5 +65,21 @@ Options currently supported by Secure Lo
 .B PRUNEFS
 and
 .B PRUNEPATHS
+
+.SH "ATTRIBUTES"
+.PP
+See \fBattributes\fR(5)
+for descriptions of the following attributes:
+.sp
+.TS
+tab(^G) allbox;
+cw(2.750000i)| cw(2.750000i)
+lw(2.750000i)| lw(2.750000i).
+ATTRIBUTE TYPE^ATTRIBUTE VALUE
+Availability^SUNWslocate
+Interface stability^Volatile
+.TE
+.sp
 .SH "SEE ALSO"
 \fBslocate\fP(1)
+...\" LSARC 2008/447 slocate for OPenSolaris
